metagraph/src/graph/representation/hash/dbg_sshash.hpp

```diff
@@ -117,6 +117,12 @@ class DBGSSHash : public DeBruijnGraph {
     size_t num_nodes_;
     Mode mode_;
 
+    void map_to_nodes_with_rc_advanced(
```

Comment:
We can remove it from the interface. I'd rather keep this as a separate function in the implementation because it might be useful later for things like mapping sequences to contigs.